.changeset/silent-buttons-peel.md

```diff
@@ -0,0 +1,20 @@
+---
+"@latticexyz/world-modules": minor
+---
+
+Adds the `ERC20Module` to `@latticexyz/world-modules`. 
```

Comment:
small thing but we should figure out what tense to use in changelogs vs commits

I usually use present tense in commits (adds) and past in changelogs (added)

Reply:
Agree with guidelines being:
- past in changelogs (added)
- [imperative present tense](https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#Commit-Guidelines) (add) in commits 😄 

we should put that down somewhere, probably the "contributing" docs page (cc @qbzzt)